extensions/BugModal/lib/WebService.pm

```diff
@@ -104,6 +105,7 @@ sub product_info {
         }
     } @{ $product->components };
     my @versions = map { { name => $_->name } } grep { $_->is_active } @{ $product->versions };
+    my @keywords = grep { $_->is_active } Bugzilla::Keyword->get_all();
```

Comment:
`@keywords` isn't used after this; please either include it in the return value or add a code comment documenting the action at a distance.